### PR TITLE
operators [N] [CI] gitops-primer (0.0.1)

### PR DESCRIPTION
**Pull request by jmontleon**
Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [X] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [X] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [X] Have you tested your Operator with all Custom Resource Definitions?
* [X] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [X] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [X] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [x] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Your submission should not

* [X] Modify more than one operator
* [X] Modify an Operator you don't own
* [X] Rename an operator - please remove and add with a different name instead
* [X] Modify any files outside the above mentioned folders
* [X] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [X] Description about the managed Application and where to find more information
2. [X] Features and capabilities of your Operator and how to use it
3. [X] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [X] Human readable name and 1-liner description about your Operator
* [x] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [X] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [x] Links to the maintainer, source code and documentation
* [X] Example templates for all Custom Resource Definitions intended to be used
* [x] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

### CODE REVIEW

**Comment by jmontleon** [comment]
@rayfordj can you double check my work. It's been awhile since I've submitted or updated an operator. And first time to the new repo.

@cooktheryan: can you let me know who else you'd like me to add in the ci.yaml as reviewers.

**Comment by cooktheryan** [comment]
@jmontleon possibly add @shawn-hurley I believe he will be leading some of these efforts in the future

**Comment by jmontleon** [comment]
We should also update the URL, even if it's just to the github repo.
https://github.com/konveyor/gitops-primer/blob/main/bundle/manifests/gitops-primer.clusterserviceversion.yaml#L370

File: operators/gitops-primer-operator/0.0.1/manifests/gitops-primer.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+    type: AllNamespaces
+  keywords:
+  - gitops
+  - operator
+  links:
+  - name: Gitops Primer
+    url: https://github.com/konveyor/gitops-primer
+  maturity: alpha
+  provider:
+    name: OCTO-ET
```
Comment (rayfordj) [inline operators/gitops-primer-operator/0.0.1/manifests/gitops-primer.clusterserviceversion.yaml:373]: @jmontleon , Is this [still] correct for the provider name or should it [now] be `Konveyor` (or `Red Hat`)?

```suggestion
  provider:
    name: Konveyor
```

Comment (rayfordj) [reply operators/gitops-primer-operator/0.0.1/manifests/gitops-primer.clusterserviceversion.yaml:373]: Looks like @cooktheryan has proposed `Konveyor` in a subsequent PR in [konveyor/gitops-primer#R360-R361](https://github.com/konveyor/gitops-primer/pull/157/files#diff-36d1afe19093400e25a5bbab91aab02bdff0a385377b7a48e629ad4208fcba11R360-R361)

**Review by rayfordj — APPROVED** [review]
/lgtm

**Comment by rayfordj** [comment]
@J0zi / @mvalarh , I believe we are settled on this PR and awaiting your ack.  Thanks!

**Pull request merged**